BlueprintUI/Sources/Blueprint View/BlueprintView.swift

```diff
@@ -43,11 +44,16 @@ public final class BlueprintView: UIView {
             if oldValue == nil && self.element == nil {
                 return
             }
+
+            logElementAssigned()
             
             setNeedsViewHierarchyUpdate()
             invalidateIntrinsicContentSize()
         }
     }
+
+    /// A name to help identify this view when profiling or debugging
+    public var name: String?
```

Comment:
> Nit: `debugIdentifier`?

Well, I see this potentially coming from a workflow screen, and I kind of want to avoid "identifier" since it's just a display thing and not used for identity checks.

Reply:
True – I guess when I see "name" I think of it as something that's potentially user-facing without a "debug" prefix.

Reply:
`debugName`?